README.md

```diff
@@ -50,6 +50,24 @@ Build the docker container (included is django, postgres, pgadmin and jupyter no
 docker-compose -f local.yml build
 ```
 
+[Create your an OpenCage API key](https://opencagedata.com/)
+
+And add your OpenCage API key to sharespots/.env/.local/.django e.g.:
```

Comment:
Do we want to prevent people putting in PRs with their personal API keys in?

Reply:
Yes, good point. I’m closing this PR. It was a bad idea of mine.

Reply:
@louiechristie Aw, I didn't mean to close the PR. As I remember Lili specifically asked for some docs on how to generate the api key on Saturday. Also the troubleshooting info looked useful too.

Reply:
Ok, I’ve reopened the PR. But we’ll have to have a think about how to to avoid people committing the api key.

I think probably we shouldn’t commit the .env .env/.local/.django file. .env folder is in the .gitignore file so it you wouldn’t expect it to be committed. But the .env/.local/.django file has already been committed before so it’s not ignored. And it’s got SECRET_KEY in it which I don’t know what that does.

A workaround might be to tell people to add the api keys to their ~/.bashrc or ~/.zshrc file.

I’m not familiar enough with Django to know the right way. We probably need to ask Lili.

Reply:
Yeah, let's see what @lili2311 thinks.

Reply:
Also thanks for explaining - I hadn't realised .envs was in .gitignore